### PR TITLE
Mage_Tax - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)
- fixedd translation in Mage_Tax_Model_System_Config_Source_Tax_Country::toOptionArray()

### CODE REVIEW

File: app/code/core/Mage/Tax/Model/Calculation.php [hunk 1]
```diff
@@ -339,7 +355,7 @@ public function getStoreRateForItem($item, $store = null)
     /**
      * Get request object for getting tax rate based on store shippig original address
      *
-     * @param   null|store $store
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Calculation.php:342]: Mage_Core_Model_Store ?

File: app/code/core/Mage/Tax/Model/Calculation.php [hunk 2]
```diff
@@ -356,10 +372,10 @@ public function getRateOriginRequest($store = null)
     /**
      * Return the default rate request. It can be either based on store address or customer address
      *
-     * @param type $store
+     * @param mixed $store
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Calculation.php:375]: Mage_Core_Model_Store ?

File: app/code/core/Mage/Tax/Model/Calculation/Rate/Title.php [hunk 3]
```diff
@@ -48,6 +49,10 @@ protected function _construct()
         $this->_init('tax/calculation_rate_title');
     }
 
+    /**
+     * @param $rateId
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Calculation/Rate/Title.php:53]: int

File: app/code/core/Mage/Tax/Model/Config.php [hunk 4]
```diff
@@ -164,7 +164,7 @@ protected function _getStoreConfig($path, $store)
     /**
      * Check if product prices inputed include tax
      *
-     * @param   mix $store
+     * @param   mixed $store
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Config.php:167]: core model store?

File: app/code/core/Mage/Tax/Model/Resource/Calculation.php [hunk 5]
```diff
@@ -459,8 +465,8 @@ public function getRateIds($request)
     /**
      * Retrieve rates by customer tax class
      *
-     * @param int $customerTaxClassId
-     * @param int $productTaxClassId
+     * @param $customerTaxClass
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Resource/Calculation.php:468]: int

File: app/code/core/Mage/Tax/Model/Resource/Calculation.php [hunk 6]
```diff
@@ -459,8 +465,8 @@ public function getRateIds($request)
...
+     * @param null $productTaxClass
```
Comment (tmotyl) [inline app/code/core/Mage/Tax/Model/Resource/Calculation.php:469]: int|null

File: app/code/core/Mage/Tax/Model/Calculation.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Calculation.php:342]: Guess there were two `@param ... $store` ... fixed mixed.

File: app/code/core/Mage/Tax/Model/Calculation.php (hunk 2 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Calculation.php:375]: `null|Mage_Core_Model_Store`?

File: app/code/core/Mage/Tax/Model/Calculation/Rate/Title.php (hunk 3 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Calculation/Rate/Title.php:53]: Comes from ...

```
        $fieldName = $this->getIdFieldName();
        if ($fieldName) {
            return $this->_getData($fieldName);
        } else {
            return $this->_getData('id');
        }
```

... should be `int`, but i guess it is `string` .... however .... changed

File: app/code/core/Mage/Tax/Model/Config.php (hunk 4 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Config.php:167]: Not in this case .... `$store` param could be more then `Mage_Core_Model_Store` here ...

`return (bool)$this->_getStoreConfig(self::CONFIG_XML_PATH_PRICE_INCLUDES_TAX, $store);`

Btw .... other untouched methods in this class use `mixed`.

File: app/code/core/Mage/Tax/Model/Resource/Calculation.php (hunk 6 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Resource/Calculation.php:469]: Don't know why i did this ...

File: app/code/core/Mage/Tax/Model/Resource/Calculation.php (hunk 5 above)
Comment (sreichel) [reply app/code/core/Mage/Tax/Model/Resource/Calculation.php:468]: Don't know why i did this ...

**Comment by colinmollenhour** [comment]
Almost there! Thanks for your patience, @sreichel!

**Pull request merged**